lib/src/utils/crypto/ffi.dart

```diff
@@ -81,23 +83,23 @@ final EVP_CIPHER_CTX_new = libcrypto.lookupFunction<
     Pointer<NativeType> Function()>('EVP_CIPHER_CTX_new');
 
 final EVP_EncryptInit_ex = libcrypto.lookupFunction<
-    Pointer<NativeType> Function(
+    IntPtr Function(
```

Comment:
can you list all the code logic use these functions?

Reply:
we need to change it or not? I dont want to impact too much to web version at the moment